manifests/plugin.pp

```diff
@@ -77,7 +77,7 @@
     $proxy_port  = undef,
 ) {
 
-  include elasticsearch
+  require elasticsearch
```

Comment:
crap, I changed it, but then squashed/reordered commits and it reappeard without /me noticing.
Now it's back as it should be.